build/akri-dotnet-containers.mk

```diff
@@ -0,0 +1,102 @@
+define add_onvif_target
```

Comment:
can this be done similarly to `add_rust_targets` so it is `add_dotnet_targets`?

Reply:
if i knew how to, i would.  really, the difference between the various add_*_targets is ONLY the differing number and values of the --build-args.

for rust, there are these 2 (with variants per platform): `--build-arg PLATFORM=$(AMD64_SUFFIX) --build-arg CROSS_BUILD_TARGET=$(AMD64_TARGET)`

for onvif, there are these 2 (with variants per platform): `--build-arg OUTPUT_PLATFORM_TAG=$(USE_OPENCV_BASE_VERSION)-$(AMD64_SUFFIX) --build-arg DOTNET_PUBLISH_RUNTIME=linux-x64`

for opcua, there are these 2 (with variants per platform): `--build-arg OUTPUT_PLATFORM_TAG=3.1-buster-slim --build-arg DOTNET_PUBLISH_RUNTIME=linux-x64`

for python, there is 1 (with variants per platform): `--build-arg PLATFORM=$(AMD64_SUFFIX)`

i'm sure there is a way to just have one add_targets, but i don't know it :(